gpkit/constraints/set.py

```diff
@@ -92,6 +97,8 @@ def __init__(self, constraints, substitutions=None):  # pylint: disable=too-many
         add_meq_bounds(self.bounded, self.meq_bounded)
 
     def __getitem__(self, key):
+        if self.idxlookup and key in self.idxlookup:
+            key = self.idxlookup[key]
```

Comment:
Really feels like this thing should be a collections.OrderedDict, but I haven't thought deeply about it